microsoft/testsuites/cvm/cvm_attestation.py

```diff
@@ -13,14 +15,15 @@
     features,
 )
 from lisa.features.security_profile import CvmEnabled
-from lisa.operating_system import Ubuntu
+from lisa.operating_system import CBLMariner, Ubuntu
```

Comment:
We should rename CBLMariner to AzureLinux. Please feel free to resolve this comment, just wanted to bring this to attention (feel free to volunteer to change in a separate PR or create an issue or notify some other maintainer :)